### PR TITLE
Do not scale SplitPane

### PR DESCRIPTION
**Pull request by raineorshine**

### CODE REVIEW

File: src/components/AppComponent.js [hunk 1]
```diff
@@ -41,9 +41,24 @@ const fontSizeLocal = +(localStorage['Settings/Font Size'] || 16)
 const tutorialLocal = localStorage['Settings/Tutorial'] === 'On'
 const tutorialStepLocal = +(localStorage['Settings/Tutorial Step'] || 1)
 
+/** A container that scales its children by the given amount. */
+const Scale = ({ amount, children }) =>
```
Comment (anmolarora1) [inline src/components/AppComponent.js]: Since this component is independent of AppComponent, maybe we can move it out

Comment (raineorshine) [reply src/components/AppComponent.js]: Agree, thanks

File: src/components/AppComponent.js [hunk 2]
```diff
@@ -41,9 +41,24 @@ const fontSizeLocal = +(localStorage['Settings/Font Size'] || 16)
...
+  <div style={{
+    transform: `scale(${amount})`,
+    transformOrigin: '0 0',
+    width: `${100 * (1 / amount)}%`
+  }}>{children}</div>
+
+/** Scaled Toolbar and Content */
+const ContentPanel = ({ scale }) =>
+  <Scale amount={scale}>
+    <Toolbar />
```
[the remark this answers is not in the document]
Comment (raineorshine) [reply src/components/AppComponent.js]: Good point

[the remark this answers is not in the document]
Comment (raineorshine) [reply src/components/AppComponent.js]: For some reason moving the `Toolbar` outside the SplitPane causes the transform to make it disappear. We'll have to address that in a separate issue.

**Pull request merged**